Triangulation_3/include/CGAL/Triangulation_hierarchy_3.h

```diff
@@ -142,21 +157,12 @@ class Triangulation_hierarchy_3
     noexcept( noexcept(Triangulation_hierarchy_3(std::move(other))) )
   {
     static_cast<Tr_Base&>(*this) = std::move(other);
-    hierarchy[0] = this;
-    for(int i=1; i<maxlevel; ++i) {
-      hierarchy[i] = other.hierarchy[i];
-      other.hierarchy[i] = nullptr;
-    }
+    hierarchy_triangulations = std::move(hierarchy_triangulations);
```

Comment:
`std::move(other.hierarchy_triangulations)` 
and
`other.init_hierarchy()` ?

Reply:
Oops, you are right, for `std::move(other.hierarchy_triangulations)`.

`this->init_hierarchy()` and `other.init_hierarchy()` are actually useless, because they were initialized correctly at the construction of the two objects, and the move operations do not change the pointers to sub-objects.